com.unity.ml-agents/Runtime/Communicator/ICommunicator.cs

```diff
@@ -11,21 +11,38 @@ internal struct CommunicatorInitParameters
         /// Port to listen for connections on.
         /// </summary>
         public int port;
+
         /// <summary>
         /// The name of the environment.
         /// </summary>
         public string name;
+
         /// <summary>
         /// The version of the Unity SDK.
         /// </summary>
-        public string version;
+        public string packageVersion;
```

Comment:
same two fields in CommunicatorInitParameters and UnityRLInitParameters. Do you think it would be okay to add a "Unity / Python" prefix to these two properties for clarity?

Reply:
How about "trainer' and "environment" to be a bit more language-agnostic?

Reply:
There is not always a trainer...
environment/remote ?

Reply:
I'm going to leave these generic in the protos (with comments), and change to Unity/python in the C# plain-old-c-sharp structs.